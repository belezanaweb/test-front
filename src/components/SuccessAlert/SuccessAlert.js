import React from 'react'
import { SuccessAlertContainer, SuccessAlertText, SuccessAlertIcon } from './SuccessAlert.styles'

export const SuccessAlert = (props) => (
  <SuccessAlertContainer>
    <SuccessAlertIcon />
    <SuccessAlertText {...props} />
  </SuccessAlertContainer>
)
