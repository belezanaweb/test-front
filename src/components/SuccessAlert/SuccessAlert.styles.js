import styled from 'styled-components'

export const SuccessAlertContainer = styled.section`
  display: grid;
  justify-items: center;
  grid-gap: 11px;
  grid-column-end: span 2;
`

export const SuccessAlertText = styled.h1`
  margin: 0;
  color: #ff7800;
  font-size: 14px;
  font-weight: 700;
  line-height: 1.21428571428571;
  text-align: center;
  text-transform: uppercase;
`

export const SuccessAlertIcon = styled.img.attrs({
  'aria-hidden': true,
  src: '/images/icon-checkmark-circle.png'
})`
  width: 40px;
  height: 40px;
  object-fit: contain;
  object-position: 50% 50%;
`
