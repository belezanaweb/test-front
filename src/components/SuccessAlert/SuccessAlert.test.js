import React from 'react'
import { render } from '@testing-library/react'
import { SuccessAlert } from './SuccessAlert'

test('renders SuccessAlert', () => {
  const { container } = render(<SuccessAlert>Teste</SuccessAlert>)

  expect(container).toMatchSnapshot()
})
