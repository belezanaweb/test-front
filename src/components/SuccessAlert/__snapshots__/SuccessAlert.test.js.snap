// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SuccessAlert 1`] = `
.c0 {
  display: grid;
  justify-items: center;
  grid-gap: 11px;
  grid-column-end: span 2;
}

.c2 {
  margin: 0;
  color: #ff7800;
  font-size: 14px;
  font-weight: 700;
  line-height: 1.21428571428571;
  text-align: center;
  text-transform: uppercase;
}

.c1 {
  width: 40px;
  height: 40px;
  object-fit: contain;
  object-position: 50% 50%;
}

<div>
  <section
    class="c0"
  >
    <img
      aria-hidden="true"
      class="c1"
      src="/images/icon-checkmark-circle.png"
    />
    <h1
      class="c2"
    >
      Teste
    </h1>
  </section>
</div>
`;
